addons/web/static/tests/views/calendar_tests.js

```diff
@@ -232,6 +232,38 @@ QUnit.module('Views', {
         calendar.destroy();
     });
 
+    QUnit.test('delete attribute on calendar doesn\'t show delete button in popover', async function (assert) {
```

Comment:
There should be a second test without the delete=0, and another one with delete=1
showing:
 the feature
 Its default
 the opposite of the feature

Reply:
I don't think we need separate test here as it is already tested in another test case, please check this: https://github.com/odoo/odoo/blob/master/addons/web/static/tests/views/calendar_tests.js#L745

Reply:
alright, did not see that